app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,12 @@
+doctype html
+html
+head
+  titleWedikoSecretSnowman/title
+  %= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' = true %
```

Comment:
This file does not seem to serve any purpose. I think it was a mistake that it was included. I will delete it.